lib/workers/pr/body/controls.ts

```diff
@@ -6,5 +6,13 @@ export async function getControls(config: BranchConfig): Promise<string> {
   const warning = (await isBranchModified(config.branchName))
     ? emojify(' :warning: **Warning**: custom changes will be lost.')
     : '';
-  return `\n\n---\n\n - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, click this checkbox.${warning}\n\n`;
+
+  const rebaseBoxUncheckedWarning =
+    config?.prRebaseBoxUnchecked === true
+      ? emojify(
+          '\n\n :memo: **Note**: The rebase/retry checkbox was unchecked because of `stopRebasingLabel` setting in your configuration'
```

Comment:
No, we shouldn't do this. If someone has the stop-rebasing label AND ticks the checkbox to rebase, we should still rebase.

Reply:
Are you sure any of these changes are necessary?

Reply:
I think it's this text you need to change instead:

![image](https://user-images.githubusercontent.com/6311784/149780371-1a84bcbd-c044-4845-989d-2501040c75ff.png)